tests/test_obsah.py

```diff
@@ -14,6 +14,10 @@ class MockApplicationConfig(obsah.ApplicationConfig):
         def playbooks_path():
             return playbooks_path.strpath
 
+        @staticmethod
+        def target_name():
+            return ['packages', 'repos']
```

Comment:
This means the Mock here has a list, while the "original" application keeps a single-item? Why?

https://github.com/theforeman/obsah/blob/43d9bfd053eed827b1468e4b54c24c7e471e9245/obsah/__init__.py#L179-L184

Reply:
I was trying to keep the original "API" in tact, are you saying I should create two mocks?